build_vm_compilers/lib/src/vm_entrypoint_builder.dart

```diff
@@ -63,16 +64,19 @@ https://github.com/dart-lang/build/blob/master/docs/faq.md#how-can-i-resolve-ski
       }
 
       var transitiveKernelModules = [
-        module.primarySource.changeExtension(vmKernelModuleExtension)
-      ].followedBy(transitiveModules.map(
-          (m) => m.primarySource.changeExtension(vmKernelModuleExtension)));
-      var appContents = <int>[];
-      for (var dependencyId in transitiveKernelModules) {
-        appContents.addAll(await buildStep.readAsBytes(dependencyId));
-      }
-      await buildStep.writeAsBytes(
+        module.primarySource.changeExtension(vmKernelModuleExtension),
+        for (var dep in transitiveModules)
```

Comment:
:tada:

Reply:
There is no end to the delight in this feature CC @munificent